src/openforms/js/components/admin/form_design/logic/actions/Actions.js

```diff
@@ -133,6 +135,32 @@ const ActionValue = ({action, errors, onChange}) => {
   );
 };
 
+const ActionVariableValue = ({action, errors, onChange}) => {
+  const allVariables = useContext(FormVariablesContext);
+
+  const getVariableChoices = variables => {
+    return variables.map(variable => [variable.key, variable.name]);
+  };
+
+  return (
+    <>
+      <DSLEditorNode errors={errors.variable}>
+        {/*TODO: This should be a searchable select for when there are a billion variables?*/}
+        <Select
+          name="variable"
+          choices={getVariableChoices(allVariables)}
+          allowBlank
+          onChange={onChange}
+          value={action.variable}
+        />
+      </DSLEditorNode>
+      <DSLEditorNode errors={errors.action?.value}>
+        <JsonWidget name="action.value" logic={action.action.value} onChange={onChange} />
+      </DSLEditorNode>
```

Comment:
Is the intention here to use JSONLogic to calculate/set the value?

edit: yep, confirmed by https://github.com/open-formulieren/open-forms/pull/1694/files#diff-087878c531b0db8b37141d5a552a674d16470c37801f1e4bdb443ee110e2aab4R133